packages/edit-site/src/components/global-styles/utils.js

```diff
@@ -4,6 +4,7 @@
 import { get, find, isString } from 'lodash';
 
 /* Supporting data. */
+export const LATEST_SCHEMA = 2; // Need to be kept in sync with WP_Theme_JSON.
```

Comment:
Could/should we link to a file reference here?

Reply:
I guess we can revert this now.